tests/integration/pause.bats

```diff
@@ -32,3 +32,42 @@ function teardown() {
   # test state of busybox is back to running
   testcontainer test_busybox running
 }
+
+@test "runc pause and resume with multi-container" {
+  # run test_busybox1 detached
+  runc run -d --console /dev/pts/ptmx test_busybox1
+  [ "$status" -eq 0 ]
+
+  wait_for_container 15 1 test_busybox1
+
+  # run test_busybox2 detached
+  runc run -d --console /dev/pts/ptmx test_busybox2
+  [ "$status" -eq 0 ]
+
+  wait_for_container 15 1 test_busybox2
+
+  # pause test_busybox1 and test_busybox2
+  runc pause test_busybox1 test_busybox2
+  [ "$status" -eq 0 ]
+
+  # test state of test_busybox1 and test_busybox2 is paused
+  testcontainer test_busybox1 paused
+  testcontainer test_busybox2 paused
+
+  # resume test_busybox1 and test_busybox2
+  runc resume test_busybox1 test_busybox2
```

Comment:
Can you include a test to make sure that if you try to pause/resume containers that don't exist the other containers still get changed (with `$? != 0`)? Just copy this test and add "nonexistant" to each `runc pause` and `runc resume` command.